sdk/tests/platform/test_az_mqtt5_policy.c

```diff
@@ -54,13 +54,16 @@ static int ref_puback = 0;
 static int ref_recv = 0;
```

Comment:
Nit: shouldn't this test be called `test_az_mosquitto5_policy.c`? It could also be general but it would need lots of `#ifdef` to switch between the various MQTT implementations we will support.

Reply:
We discussed that the tests in the platform folder would be integration tests in the same way we have the ones for POSIX and Win32 (future implementation).